packages/graphile-build-pg/src/plugins/PgIntrospectionPlugin.js

```diff
@@ -139,393 +205,550 @@ function readFile(filename, encoding) {
   });
 }
 
-export default (async function PgIntrospectionPlugin(
-  builder,
-  {
-    pgConfig,
-    pgSchemas: schemas,
-    pgEnableTags,
-    persistentMemoizeWithKey = (key, fn) => fn(),
-    pgThrowOnMissingSchema = false,
-    pgIncludeExtensionResources = false,
-  }
+/****************************************************************************************
+ * Build Extensions - Add two ways to query the PostgreSQL introspection results to the
+ * build object; one by 'kind' and the other by 'id'. See below for details
```

Comment:
I think this text is out of date; this PR should introduce no API changes since it is only a refactor, and thus it should (and does?) only expose the `pgIntrospectionResultsByKind`